test/consumer.ts

```diff
@@ -134,7 +134,7 @@ describe('Consumer', () => {
 
       consumer.start();
 
-      const err = await pEvent(consumer, 'error');
+      const err: any = await pEvent(consumer, 'error');
```

Comment:
Why add a type declaration?

Reply:
after the update the ts-types I started to have issues on this test because the types... feel free to update it.

Reply:
Hey @nspragg  I had some issues on my configs, I fix it now and could remove the any from the tests... all good!